<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="welcomeL">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QLabel" name="ql">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose player 1 teams :</string>
    </property>
   </widget>
   <widget class="QLabel" name="ipL">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>IP Address : </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="IranRB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Iran</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="brasilRB">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Brasil</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>280</y>
      <width>97</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="argentinaRB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Argentina</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QLineEdit" name="ipAddress">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>211</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>97</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="germanyRB">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>170</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Germany</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="spainRB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Spain</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="italyRB">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>230</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Italy</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player1BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="argentinaRB_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Argentina</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player2BG</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="ql_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose player 2 teams :</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="germanyRB_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Germany</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player2BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="italyRB_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Italy</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player2BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="brasilRB_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>200</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Brasil</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player2BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="spainRB_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>117</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Spain</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player2BG</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="IranRB_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Iran</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">player2BG</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="player2BG"/>
  <buttongroup name="player1BG"/>
 </buttongroups>
</ui>
